ci/requirements-py3.10-pandas1.5.3-pydantic1.10.11.txt

```diff
@@ -13,7 +13,6 @@ astroid==2.15.8
 asttokens==2.4.1
     # via stack-data
 asv==0.6.3
-    # via -r /var/folders/wd/sx8dvgys011_mrcsd1_vrz1m0000gn/T/tmp6ejs7w6z
```

Comment:
I just removed those, as they are not meaningful, ok?

Reply:
if these are being removed manually, this will have no effect when ci/dev dependencies are regenerated

Reply:
I just replaced the entire line using VS Code's regular expression search, I thought that it was some dirty from my setup by I just noted that the `main` branch already contains those.

Reply:
a better solution for this would be to update the `uv pip compile` command [here](https://github.com/unionai-oss/pandera/blob/main/noxfile.py#L356-L365) and [here](https://github.com/unionai-oss/pandera/blob/main/noxfile.py#L373-L382) with the `--no-annotate`  flag

Reply:
In the application I work on, these annotations are very useful to understand which package is currently limiting an indirectly dependency version.
I implemented the suggested changes, just let me know which version you want to keep :)